scripts/ci/update_ext_cmd_tree.py

```diff
@@ -44,8 +44,7 @@ def update_cmd_tree(ext_name):
     az_cli.invocation = invoker
 
     sys.path.append(ext_dir)
-    extension_command_table, _ = _load_extension_command_loader(invoker.commands_loader,
-                                                                "", ext_mod)
+    extension_command_table, _ = _load_extension_command_loader(invoker.commands_loader, None, ext_mod)
```

Comment:
`""` empty string will cause cut when loading command table for aaz commands. Replace it by `None` for fully load.

Reply:
Could you give some more info? I can't really understand the context. 😅